config/initializers/sentry.rb

```diff
@@ -1,6 +1,19 @@
 return unless ENV['SENTRY_DSN'].present?
 
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = ENV['SENTRY_DSN']
-  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+
+
+  filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters.map(&:to_s))
+  config.before_send = lambda do |event, hint|
+    event.request.data = filter.filter(event.request.data)
+    event
+  end
+
+  # This is the default value for this option, putting here for visiblity
+  # This will remove the request body from the information sent to sentry
+  config.send_default_pii = false
```

Comment:
Just thinking about this, maybe we could have the PII on for dev/staging/uat but off on pre-prod / production?
```
config.send_default_pii = Rails.env.development? || ENV.fetch('HOSTNAME', 'UNKNOWN') =~ /(\-(dev|staging|uat)\-)/i
```

Reply:
Yes, that's what @smoothcontract was suggesting in the ticket. I was going to do that as a separate PR after this one got merged, just to keep the decision encapsulated in one small change for easy retrieval later if needed :)

Reply:
jolly good 👍